src/AppInstallerCommonCore/Public/winget/FolderFileWatcher.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "pch.h"
+
+namespace AppInstaller::Utility
+{
+    // Watch for new/renamed files recursively in a given directory.
+    struct FolderFileWatcher
+    {
+        FolderFileWatcher(const std::filesystem::path& path);
+        ~FolderFileWatcher();
+
+        FolderFileWatcher(const FolderFileWatcher&) = delete;
+        FolderFileWatcher& operator=(const FolderFileWatcher&) = delete;
+
+        FolderFileWatcher(FolderFileWatcher&&) = delete;
+        FolderFileWatcher& operator=(FolderFileWatcher&&) = delete;
+
+        void start();

```

Comment:
```suggestion
        void Start();
```

Uppercase function names.